app/views/analytics/index.html.erb

```diff
@@ -153,19 +147,50 @@
     </div>
   </div><!-- row   -->
 
-  <!-- Comments List-->
   <div class="row">
-    <div class="col-sm-12">
+  <!-- Editors List-->
+    <div class="col-sm-4">
       <div class="chart-wrapper">
         <div class="chart-title background-orange">
+          Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+        </div>
+        <div class="chart-stage">
+         <table class='table leader-list'>
+           <tr>
+            <th>Editor</th>
+            <th>Updates</th>
+           </tr>
+
+           <% case_updaters_count(10).each do |who| %>
+            <tr>
```

Comment:
It might be useful to add a couple of more lines before `<tr>` that assign the different elements of `who` to more distinct variables.  So something like:

`
<% case_updaters_count(10).each do |who| %>
<% user_id = who[0] %>
<% user_object = who[1] %>
<tr>
`